src/hotspot/share/gc/z/zHeap.cpp

```diff
@@ -350,6 +351,9 @@ bool ZHeap::mark_end() {
   // Process weak roots
   _weak_roots_processor.process_weak_roots();
 
```

Comment:
line 341 MetaspaceGC::compute_new_size(); is moved to _unload.finish(). What's the reason we still leave it here?